addons/binding/org.openhab.binding.omnikinverter/build.properties

```
@@ -0,0 +1,10 @@
+source.. = src/main/java/,\
+           src/test/java/,\
+           src/test/resources
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               about.html
```

```suggestion
               NOTICE
```

See also the [community topic](https://community.openhab.org/t/tutorial-update-license-to-epl-v2-0-of-your-binding-in-development-or-pull-request/65088?u=wborn).

---

Is the build.properties still needed for 2.5.0 ?

---

You can remove it because these resources are nowadays automatically added.